lib/Cargo.toml

```diff
@@ -43,3 +43,6 @@ clap = "2.33.3"
 indoc = "1.0.3"
 sh-inline = "0.1.0"
 structopt = "0.3.21"
+
+[package.metadata.docs.rs]
+features = ["dox"]
```

Comment:
I think you will have to chain this to the other dependencies, like https://github.com/gtk-rs/gtk-rs-core/blob/0.14.2/gio/Cargo.toml#L31.

Reply:
Thanks, fixed!